command_server/command_server.py

```diff
@@ -37,7 +38,9 @@ def __init__(self, host=None, port=None, loop=None):
         logging.info('Init Server on host %s:%s' % (host, port))
         self._loop = loop or asyncio.get_event_loop()
         #TODO: MOve debug mode to config file.
-        self._server = self._loop.create_server(lambda: HttpRequestHandler(),
+        self._app = web.Application(loop=loop)
```

Comment:
Ofc we don't use HttpRequestHandler anymore, it is left as a memory of old approach). I decide to use web.Application as it more common and suitable approach. HttpRequestHandler haven't good router, and it not very common: actually it can't be applicable for using Websokets and Streams, and large responses, because it finalize response by `response.write_eof()`. So I decide move this logic for specific views